src/components/Common/Filters/Filters.vue

```diff
@@ -53,7 +53,13 @@
         </div>
       </tabs>
     </div>
+
+    <div class="card-panel orange lighten-3" v-show="currentFilter.active">
+      <span>Warning: There is actually an active filter, not every documents are shown.</span>
```

Comment:
The way I understand it, you meant `currently` and not `actually`, which is a false friend.

Here is a change proposal, I let you decide how you want to deal with this.

```suggestion
      <span>Warning: a filter has been set, some documents might be hidden.</span>
```